### PR TITLE
Added Bugfix

### PR DESCRIPTION
**Pull request by light-orth**
Jonathan Orth's bugfix for Dark Reader

### CODE REVIEW

File: _posts/2024-05-05-Jonathan-O-Darkreader-Fix.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+# title:  "Sample Blog Post"
+
+# Uncomment only one of the below categories
+categories: 
+Bug Fix
```
Comment (MoralCode) [inline _posts/2024-05-05-Jonathan-O-Darkreader-Fix.md:10]:
```suggestion
- Bug Fix
```
small metadata typo is causing jekyll to freak out

Comment (light-orth) [reply _posts/2024-05-05-Jonathan-O-Darkreader-Fix.md]: Thank you, should be fixed now.

**Comment by MoralCode** [comment]
Looks good! Its not typically necessary/customary to have trailing slashes after image paths (i.e. `darkreader mascot`), but it doesn't seem to be breaking anything so i'll go ahead and merge.

Thanks!

**Comment by light-orth** [comment]
Oh ok thank you, but looking at the posted blog it does seem like it's causing issues with the image's visibility after all, so I can go ahead and correct it.

**Pull request merged**